bears/ruby/RubyFlogBear.py

```diff
@@ -0,0 +1,56 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+from coalib.results.Result import Result
+from coalib.results.SourceRange import SourceRange
+
+
+@linter(executable='flog', use_stdin=True)
+class RubyFlogBear:
+    """
+    Check Ruby files for tortured code and reports in an easy to read
+    pain report.
+    """
+    LANGUAGES = {'Ruby'}
+    REQUIREMENTS = {GemRequirement('flog', '4.6.1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    SEE_MORE = 'https://github.com/seattlerb/flog'
+
+    def create_arguments(self, filename, file, config_file, threshold: int=1):
+
+        args = ('', filename)  # ('-t', str(threshold), filename)
+        self.log(20, args)
```

Comment:
why 20? I'm pretty sure the linter already logs the args at debug level, this shouldn't be needed